testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -285,7 +285,7 @@ public void deleteAllCookiesForMasterRealm() {
     protected void deleteAllCookiesForRealm(String realmName) {
         // we can't use /auth/realms/{realmName} because some browsers (e.g. Chrome) apparently don't send cookies
         // to JSON pages and therefore can't delete realms cookies there; a non existing page will do just fine
```

Comment:
Please update the comment to explain which browsers require an existing page

```suggestion
        // to JSON pages and therefore can't delete realms cookies there. Apparently it needs to be an existing page because ...
```

Reply:
Thanks, but I prefer waiting for @mposolda for a proper review and checking if that comment still makes sense.